ui/chapter/Intro.tsx

```diff
@@ -1,14 +1,26 @@
+import clsx from 'clsx'
+
 import { Title } from 'ui'
 
 export default function Intro({
   children,
   heading,
+  mobileCentered,
```

Comment:
Not a huge fan of props like these, makes it hard to override any css classes that are being added if this value is `true`. Let's instead just add an optional `className` prop so from the parent we can pass `my-auto my-8` if/when needed.